doc/examples/tls.rst

```diff
@@ -249,3 +249,14 @@ revocation checking failed::
   [('SSL routines', 'tls_process_initial_server_flight', 'invalid status response')]
 
 See :ref:`OCSP` for more details.
+
+Python 3.10+ is incompatible with TLS/SSL on MongoDB <= 3.4
```

Comment:
On Windows. It seems to work just fine everywhere else.

Reply:
I tested locally on macos and it's the same error behavior due to no shared ciphers.

Reply:
Thanks for checking.